runner/src/main/java/org/ananas/runner/legacy/steps/files/TextConnector.java

```diff
@@ -19,7 +19,7 @@
   public TextConnector(
       String stepId, String url, Pipeline pipeline, boolean doSampling, boolean isTest) {
     super(StepType.Connector);
-    final Schema schema = Schema.builder().addField("text", Schema.FieldType.STRING).build();
+    final Schema schema = Schema.builder().addStringField("text").build();
```

Comment:
Please move this connector from `legacy.files` to `steprunner.files` package.

Reply:
acceptance test are added

Reply:
this TextConnector is not used. I refactored it but it's still legacy.